docs/examples/101/sql-database/main.bicep

```diff
@@ -0,0 +1,47 @@
+param serverName string {
+  metadata: {
+    description: 'The name of the SQL logical server.'
```

Comment:
nit and personal preference - not sure how descriptive these descriptions are given the param names are pretty self explanatory. They end up adding a lot of lines of code for not much benefit.

Reply:
Removed.